setup.py

```diff
@@ -10,7 +10,7 @@
 from distutils.spawn import find_executable
 from distutils import log
 from platform import architecture
-from subprocess import Popen, CalledProcessError, PIPE, check_call
+from subprocess import Popen, CalledProcessError, PIPE, check_call, check_output
```

Comment:
@denfromufa Did you try using [`_check_output`](https://github.com/pythonnet/pythonnet/blob/573f90367137b30519a6b204bed7fafc7b6877ef/setup.py#L282) instead?